rdrf/rdrf/users/views.py

```diff
@@ -86,3 +155,5 @@ def post(self, request, *args, **kwargs):
         if request.user == self.user:
             auth.logout(request)  # Force logout, requiring the user to reauthenticate with their new email address
```

Comment:
Wondering if I should do this instead: https://github.com/django/django/blob/107865780aa44914e21d27fdf4ca269bc61c7f01/django/contrib/auth/views.py#L369

Reply:
Yeah, seems to be recommended in the docs:

https://docs.djangoproject.com/en/3.2/topics/auth/default/#session-invalidation-on-password-change

Reply:
This only works if the password value has changed.
https://docs.djangoproject.com/en/2.2/topics/auth/customizing/#django.contrib.auth.models.AbstractBaseUser.get_session_auth_hash